test/Autofac.Extras.DynamicProxy.Test/Autofac.Extras.DynamicProxy.Test.csproj

```diff
@@ -44,6 +44,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.6.0" />
+    <PackageReference Include="Moq" Version="4.20.70" />
```

Comment:
Can we use NSubstitute here instead of Moq? I am a fan of supporting open source, but some of the phone-home stuff that was implemented leaves Moq on vulnerability lists for some secure enterprises. In places where we _already had_ Moq, it should be pinned to the version just before that was implemented; for tests that are adding new mocking frameworks, we should go NSubstitute.